tests/tailer/savedown_withcount/settings.q

```diff
@@ -0,0 +1,9 @@
+// IPC connection parameters
+.servers.CONNECTIONS:`segmentedtickerplant`sort`gateway`rdb`hdb`tailer_seg1`tr_seg1
+.servers.USERPASS:`admin:admin;
```

Comment:
the test proc has the `tailer` proctype, so surely it should be automatically using the tailer username & pass? i.e. this line shouldn't be needed I don't think